### ISSUE TITLE
Your inference results are good, but mine is bad

### ISSUE THREAD

**Issue by namoamituo** [tony-laoshi/tony-laoshi.github.io#1]
First of all, thanks for your outstanding work on anynet. The effect I get from using the pre-trained model of anynet is very poor, however, the effect you get from the Onnx inference model is very good. I wonder what you have done on the  anynet pre-trained model.I would appreciate it if you could answer my question.

**Comment by tony-laoshi** [comment]
> First of all, thanks for your outstanding work on anynet. The effect I get from using the pre-trained model of anynet is very poor, however, the effect you get from the Onnx inference model is very good. I wonder what you have done on the anynet pre-trained model.I would appreciate it if you could answer my question.

If you exported the **ONNX** model, i mean the **anynet.onnx**  without problems, then my guess is that your test demo when you used **ONNX Runtime** for inference. You can see my examples [here ](https://github.com/tony-laoshi/AnyNet/tree/master/onnx%20test) and there are the exported onnx model as well.  hope it helps you.

**Comment by namoamituo** [comment]
Can I communicate with you in Chinese

**Comment by tony-laoshi** [comment]
> Can I communicate with you in Chinese

As you wish.  当然你也要坚持你导出的模型是否有问题。

**Comment by tony-laoshi** [comment]
> Can I communicate with you in Chinese

As you wish.  当然你也要确认你导出的模型是否正确。

**Comment by namoamituo** [comment]
万分感谢您的回答！实际上，我是使用您的scripts/inference.py以及model/anynet2.onnx进行推理的，您的inference.py中没有可视化的部分，我使用cv2.imshow()将disp1,disp2显示出来，效果很差。另外，我使用mileyan/AnyNet的预训练模型，在本地使用kitti2012继续微调之后，测试的效果也很差。我想知道您在将原模型导出为onnx模型之前，测试的效果好吗？如果好的话，想知道您是做了一些什么环节（我忽略了哪些环节）。

**Comment by namoamituo** [comment]
Sorry to bother you again,Can you share your visualization code with me?Thanks a lot.

**Comment by tony-laoshi** [comment]
> Sorry to bother you again,Can you share your visualization code with me?Thanks a lot.

[here](https://github.com/tony-laoshi/AnyNet/tree/master/onnx%20test/scripts) you are.   !! you will swith the onnx model which i used **anynet_withCVA.onnx**  to **anynet2.onnx**. Of course all the relate path.

**Comment by namoamituo** [comment]
Thank you for your guidance, I would like to know if the onnx model you exported does not contain spn module

**Comment by tony-laoshi** [comment]
> 

you are right. My exported onnx model does not contain SPN module. Because the original SPN module requires cuda support.   The source code has also been changed to enable the SPN module to run without CUDA. You can look at this [repository](https://github.com/BangwenHe/AnyNet)

**Comment by namoamituo** [comment]
Do you mean that the Onnx model exported by He's repository includes the spn module?
